host_vars/ritchie/prometheus.yml

```diff
@@ -0,0 +1,35 @@
+---
+prometheus_configuration:
+  global:
+    scrape_interval: 15s  # Set the scrape interval to every 15 seconds. Default is every 1 minute.
+    evaluation_interval: 15s  # Evaluate rules every 15 seconds. The default is every 1 minute.
+    # scrape_timeout is set to the global default (10s).
+
+  # Alertmanager configuration
+  alerting:
+    alertmanagers:
+      - static_configs:
+          - targets: []
+
+  rule_files:
+    # - "first_rules.yml"
+    # - "second_rules.yml"
+
+  scrape_configs:
```

Comment:
Would this be where we add other services that need to be scraped?

When we get the VPN up and running, is this where we'd add the services running on other machines too?

Reply:
Yes